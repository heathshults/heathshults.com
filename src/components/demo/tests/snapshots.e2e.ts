import { snapIt } from '../../../test';

const component = 'demo';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('renders correctly with no props', '<hs-demo></hs-demo>');
    snap(
      'renders a badge component and a code area',
      `<hs-demo language="html" code='<hs-badge type="success">YO!</hs-badge>'></hs-demo>`
    );
  });
});
