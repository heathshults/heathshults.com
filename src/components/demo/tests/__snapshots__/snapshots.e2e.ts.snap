// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`demo renders renders a badge component and a code area 1`] = `"<hs-demo language=\\"html\\" code=\\"<hs-badge type=&quot;success&quot;>YO!</hs-badge>\\" class=\\"hydrated\\"><div aria-label=\\"Demo component\\" class=\\"demo-markup-source u-letter-box-small undefined\\"><hs-badge type=\\"success\\" class=\\"hydrated\\"><span class=\\"hs-badge hs-badge--success\\">YO!</span></hs-badge></div><div class=\\"u-letter-box-small\\"><pre class=\\"u-pre\\"><code aria-hidden=\\"\\" tabindex=\\"-1\\" class=\\"u-code u-code--multiline html\\" contenteditable=\\"\\">&lt;hs-badge type=\\"success\\"&gt;YO!&lt;/hs-badge&gt;</code></pre></div></hs-demo>"`;

exports[`demo renders renders correctly with no props 1`] = `"<hs-demo class=\\"hydrated\\"><div aria-label=\\"Demo component\\" class=\\"demo-markup-source u-letter-box-small undefined\\"></div><div class=\\"u-letter-box-small\\"><pre class=\\"u-pre\\"><code aria-hidden=\\"\\" tabindex=\\"-1\\" class=\\"u-code u-code--multiline\\" contenteditable=\\"\\"></code></pre></div></hs-demo>"`;
